-------------Error at 04/26/2020 10:41:41------------- 
 Stump.Server.BaseServer.Plugins.PluginManager.LoadAllPlugins -> 
	Cannot load unexistant plugin path ./plugins/ 
-------------Error at 04/26/2020 10:41:41------------- 

-------------Error at 04/26/2020 10:43:16------------- 
 Stump.Server.BaseServer.ServerBase.KeepSQLConnectionAlive -> 
	Cannot ping SQL connection : {0} 
-------------Error at 04/26/2020 10:43:16------------- 

-------------Warn at 04/26/2020 10:43:16------------- 
 Stump.Server.BaseServer.ServerBase.KeepSQLConnectionAlive -> 
	Try to Re-open the connection 
-------------Warn at 04/26/2020 10:43:16------------- 

-------------Error at 04/26/2020 10:43:45------------- 
 Stump.Core.Threading.SelfRunningTaskPool.ProcessCallback -> 
	Exception raised when processing TimerEntry Stump.Core.Timers.TimedTimerEntry (Callback = Stump.Server.AuthServer.Managers.AccountManager.Void RefreshBans(), Delay = 60000) in Stump.Server.BaseServer.Benchmark.BenchmarkedTaskPool : System.InvalidOperationException: Connection must be valid and open.
   à MySql.Data.MySqlClient.Interceptors.ExceptionInterceptor.Throw(Exception exception)
   à MySql.Data.MySqlClient.MySqlConnection.Throw(Exception ex)
   à MySql.Data.MySqlClient.MySqlCommand.Throw(Exception ex)
   à MySql.Data.MySqlClient.MySqlCommand.CheckState()
   à MySql.Data.MySqlClient.MySqlCommand.ExecuteReader(CommandBehavior behavior)
   à MySql.Data.MySqlClient.MySqlCommand.ExecuteDbDataReader(CommandBehavior behavior)
   à System.Data.Common.DbCommand.System.Data.IDbCommand.ExecuteReader()
   à Stump.ORM.Database.<Query>d__71`1.MoveNext() dans C:\Users\Ordinateur\Desktop\Decay 2.51 - Final\Core\Stump.ORM\PetaPoco.cs:ligne 783
   à System.Collections.Generic.List`1.InsertRange(Int32 index, IEnumerable`1 collection)
   à System.Collections.Generic.List`1.AddRange(IEnumerable`1 collection)
   à Stump.Server.AuthServer.Managers.AccountManager.RefreshBans() dans C:\Users\Ordinateur\Desktop\Populus 2.51\server\Stump.Server.AuthServer\Managers\AccountManager.cs:ligne 96
   à Stump.Core.Timers.TimedTimerEntry.Trigger() dans C:\Users\Ordinateur\Desktop\Decay 2.51 - Final\Core\Stump.Core\Timers\TimedTimerEntry.cs:ligne 128
   à Stump.Core.Threading.SelfRunningTaskPool.ProcessCallback(Object state) dans C:\Users\Ordinateur\Desktop\Decay 2.51 - Final\Core\Stump.Core\Threading\SelfRunningTaskPool.cs:ligne 223. 
-------------Error at 04/26/2020 10:43:45------------- 

-------------Error at 04/26/2020 10:43:45------------- 
 Stump.Core.Threading.SelfRunningTaskPool.ProcessCallback -> 
	Exception raised when processing TimerEntry Stump.Core.Timers.TimedTimerEntry (Callback = Stump.Server.AuthServer.Managers.AccountManager.Void RefreshBans(), Delay = 60000) in Stump.Server.BaseServer.Benchmark.BenchmarkedTaskPool : System.InvalidOperationException: Connection must be valid and open.
   à MySql.Data.MySqlClient.Interceptors.ExceptionInterceptor.Throw(Exception exception)
   à MySql.Data.MySqlClient.MySqlConnection.Throw(Exception ex)
   à MySql.Data.MySqlClient.MySqlCommand.Throw(Exception ex)
   à MySql.Data.MySqlClient.MySqlCommand.CheckState()
   à MySql.Data.MySqlClient.MySqlCommand.ExecuteReader(CommandBehavior behavior)
   à MySql.Data.MySqlClient.MySqlCommand.ExecuteDbDataReader(CommandBehavior behavior)
   à System.Data.Common.DbCommand.System.Data.IDbCommand.ExecuteReader()
   à Stump.ORM.Database.<Query>d__71`1.MoveNext() dans C:\Users\Ordinateur\Desktop\Decay 2.51 - Final\Core\Stump.ORM\PetaPoco.cs:ligne 783
   à System.Collections.Generic.List`1.InsertRange(Int32 index, IEnumerable`1 collection)
   à System.Collections.Generic.List`1.AddRange(IEnumerable`1 collection)
   à Stump.Server.AuthServer.Managers.AccountManager.RefreshBans() dans C:\Users\Ordinateur\Desktop\Populus 2.51\server\Stump.Server.AuthServer\Managers\AccountManager.cs:ligne 96
   à Stump.Core.Timers.TimedTimerEntry.Trigger() dans C:\Users\Ordinateur\Desktop\Decay 2.51 - Final\Core\Stump.Core\Timers\TimedTimerEntry.cs:ligne 128
   à Stump.Core.Threading.SelfRunningTaskPool.ProcessCallback(Object state) dans C:\Users\Ordinateur\Desktop\Decay 2.51 - Final\Core\Stump.Core\Threading\SelfRunningTaskPool.cs:ligne 223. 
-------------Error at 04/26/2020 10:43:45------------- 

-------------Error at 04/26/2020 10:45:38------------- 
 Stump.Server.BaseServer.ServerBase.KeepSQLConnectionAlive -> 
	Cannot ping SQL connection : {0} 
-------------Error at 04/26/2020 10:45:38------------- 

-------------Warn at 04/26/2020 10:45:38------------- 
 Stump.Server.BaseServer.ServerBase.KeepSQLConnectionAlive -> 
	Try to Re-open the connection 
-------------Warn at 04/26/2020 10:45:38------------- 

-------------Error at 04/26/2020 10:47:44------------- 
 Stump.Server.AuthServer.IPC.IPCClient.SendError -> 
	IPC error : {0} 
-------------Error at 04/26/2020 10:47:44------------- 

-------------Error at 04/26/2020 10:50:31------------- 
 Stump.Server.AuthServer.IPC.IPCClient.SendError -> 
	IPC error : {0} 
-------------Error at 04/26/2020 10:50:31------------- 

-------------Error at 04/26/2020 10:53:22------------- 
 Stump.Server.AuthServer.IPC.IPCClient.SendError -> 
	IPC error : {0} 
-------------Error at 04/26/2020 10:53:22------------- 

-------------Error at 04/26/2020 10:55:56------------- 
 Stump.Server.AuthServer.IPC.IPCClient.SendError -> 
	IPC error : {0} 
-------------Error at 04/26/2020 10:55:56------------- 

-------------Error at 04/26/2020 10:58:39------------- 
 Stump.Server.AuthServer.IPC.IPCClient.SendError -> 
	IPC error : {0} 
-------------Error at 04/26/2020 10:58:39------------- 

-------------Error at 04/26/2020 10:59:01------------- 
 Stump.Server.BaseServer.Plugins.PluginManager.LoadAllPlugins -> 
	Cannot load unexistant plugin path ./plugins/ 
-------------Error at 04/26/2020 10:59:01------------- 

-------------Error at 04/26/2020 14:18:41------------- 
 Stump.Server.BaseServer.Plugins.PluginManager.LoadAllPlugins -> 
	Cannot load unexistant plugin path ./plugins/ 
-------------Error at 04/26/2020 14:18:41------------- 

-------------Error at 04/26/2020 15:49:04------------- 
 Stump.Server.BaseServer.Plugins.PluginManager.LoadAllPlugins -> 
	Cannot load unexistant plugin path ./plugins/ 
-------------Error at 04/26/2020 15:49:04------------- 

